docs-old/Documentation.md

```diff
@@ -442,6 +443,26 @@ let a = [1;2;3]
 let b = [|4;5;6|]
 ```
 
+### fsharp_max_if_then_short_width
```

Comment:
@yisusalanpng keep in mind that I added this documentation bit to the old docs.
Please move it as well in https://github.com/fsprojects/fantomas/pull/2336